rfc/2020/scale-testing/kohsheen1234/rfc.md

```diff
@@ -0,0 +1,53 @@
+- Contribution Name: Scale Testing
+- Implementation Owner:  ['kohsheen1234'](https://github.com/kohsheen1234)
+- Start Date: 29-02-2020
+- Target Date: 
+- RFC PR:
+- Linkerd Issue: [linkerd/linkerd2#3895](https://github.com/linkerd/linkerd2/issues/3895)
+- Reviewers: 
+
+[summary]: #summary
+
+# Summary
+This conrtibution aims to completely automate the scale tests, so that scale-testing can be performed very easily. Automating scale tests is done to ensure the performance of Linkerd is maintained.
```

Comment:
s/automate the scale tests/automate the scale test framework
s/scale-testing/scale testing

> Automating scale tests is done to ensure the performance of Linkerd is maintained.

The reason why we want to automate the scale test framework is to ensure the tests are repeatable by the community. The test suite doesn't _maintain_ the performance of Linkerd. Its serves to provide visibility into the proxy's performance and reveal any performance degration.